src/subscript/welltest_dpds/welltest_dpds.py

```diff
@@ -0,0 +1,475 @@
+#!/usr/bin/env python
+
+import sys
+import os
+import argparse
+import numpy as np
+from ecl.summary import EclSum
+
+
+DESCRIPTION = """
+Script to extract simulated welltest results from simulator output.
+Typically used to compare with output from a welltest analysis, eg using Cappa
+
+Required summary vectors in sim deck:
+  * wbhp:well_name
+  * wopr:well_name if phase == OIL
+  * wgpr:well_name if phase == GAS
+
+Outputs the following files:
+  * dpdspt_lag1 eg superpositioned time derivative of pressure lag 1
+  * dpdspt_lag2 eg superpositioned time derivative of pressure lag 2
+  * spt superpositioned time
+
+according to the naming convention; outputdirectory/key_outfilesuffix.csv
+
+And a uinfied csv file with the following vectors:
+  * cum time
+  * wbhp vs cum time
+  * wopr vs cum time
+  * wgpr vs cum time
+  * wwpr vs cum time
+
+according to the naming convention; outputdirectory/welltest_output_outfilesuffix.csv
+"""
+
+"""
+Script is a rewrite of a legacy script originally developed and improved by:
+ * Jon Sætrom
+ * Bjørn Kåre Hegstad
+ * Cecile Otterlei
+ * Hodjat Moradi
+
+AUTHOR:
+ * Eivind Smørgrav
+
+To do:
+ * Support pseudo pressure vs time relevant for gas and gas condensate fields.
+ * Add cli option to output other vectors in csv file
+
+:meta private:
+"""
+
+CATEGORY = "modelling.reservoir"
+
+EXAMPLES = """
+.. code-block:: console
+
+ FORWARD_MODEL WELLTEST_DPDS(<ECLBASE>, <WELLNAME>=DST_WELL)
+
+ FORWARD_MODEL  WELLTEST_DPDS(<ECLBASE>, <WELLNAME>=OP_1, <PHASE>=GAS,
+    <BUILDUP_NR>=1, <OUTPUTDIRECTORY>=dst, <OUTFILESSUFIX>=OP_1_1)
+
+"""
+
+
+def get_parser():
+    """
+    Define the argparse parser
+
+    Returns:
+        parser (argparse.ArgumentParser)
+    """
+    parser = argparse.ArgumentParser(
+        description=DESCRIPTION,
+        formatter_class=argparse.RawTextHelpFormatter,
+    )
+
+    parser.add_argument(
+        "eclcase",
+        type=str,
+        help="Eclipse case to extract from",
+    )
+    parser.add_argument(
+        "wellname",
+        type=str,
+        help="Name of well to extract results from",
+    )
+    parser.add_argument(
+        "--outfilessuffix",
+        type=str,
+        help='Suffix to be added to result files. Default: ""',
+        default="",
+    )
+    parser.add_argument(
+        "-n",
+        "--buildup_nr",
+        type=int,
+        help="Buildup nr, eg which buildup to extract. Counting from 1. Default: 1 ",
+        default=1,
+    )
+    parser.add_argument(
+        "-o",
+        "--outputdirectory",
+        type=str,
+        help="Directory to put the output files. Detault: .",
```

Comment:
The default is already provided in the `default` argument. To get defaults values into `welltest_dpds --help` there is another trick

Reply:
https://github.com/equinor/subscript/blob/3f06533e0d1ce94c9ab6ab0996c8beabf794829e/src/subscript/presentvalue/presentvalue.py#L34